homepage/blocks/homepage-brick/homepage-brick.css

```
@@ -202,7 +202,7 @@ body:not([data-mep-highlight="true"]) .section.masonry [data-removed-manifest-id
 }
 
 .homepage-brick p,
-.homepage-brick [class^="body-"] { margin: var(--spacing-xxs) 0; }
+.homepage-brick [class^="body-"] { margin: 0 0 var(--spacing-xxs) 0; }
```

```suggestion
.homepage-brick [class^="body-"] { margin-bottom: var(--spacing-xxs); }
```
maybe we just update to only change margin-bottom?